.changeset/ninety-insects-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/typescript-react-query': major
+---
+
+Queries without variables will no longer have an undefined entry in their query key
```

Comment:
Wouldn't this be a bug fix instead of a major change?

Reply:
Sorry, I was on vacation last week.

It depends on how you look at it. If you rely on the `undefined` entry, this might invalidate your query after this change, which would make it a breaking change. I would sill say it is a bug, but was not sure if anyone relies on the current behavior. I can change it to bugfix and make it a minor change if you prefer it though.